index.html

```diff
@@ -250,6 +250,36 @@ <h2 class="headeraboutme">About me</h2>
       </ul>
     </div>
   </Section>
+
+  <section class="contactme">
+      <h4 class="always">I am always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+    <form action="https://formspree.io/f/xvonwjkn" method="post" class="contactform">
+      <label for="name" class="name-label" ></label>
+      <input type="text" class="name" name="name" id="name" maxlength="30" placeholder="Full name" required>
+      <hr class="division division-line2"></hr>
+      <label for="mail" class="mail-label"></label>
+      <input type="email" class="email" name="email" id="mail" placeholder="Email address" required> 
+      <hr class="division division-line2"></hr>
+      <label for="msg" class="message-label"></label>
+      <textarea class="message" id="msg" maxlength="500" placeholder="Write me something..." required></textarea>
+      <div class="container-touch">
+      <button class="button1">Get in touch</button>
+      </div>
+    </form>
```

Comment:
Great job following the design of the Figma template provided. 🔥 However there is a tiny change I would recommend you make. 

- [ ] Please make sure you stick to the design of the Figma template as closely as possible. This trait will help you satisfy your future clients and gain a reputation in the development industry. 🧰 

Currently, your margins for the forms and submit button do not match the Figma design kindly check below, compare, and correct the difference. ♻️ 

### Figma design ⬇️ 
![image](https://github.com/eamador1/portfolio-project/assets/117144028/5c54e886-beb3-40a1-90ac-97de2c555646)


### Your design ⬇️ 
![image](https://github.com/eamador1/portfolio-project/assets/117144028/cfbfd0fd-8a1a-41a6-ba4b-027d4daa681b)